pkgs/tools/misc/duf/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, buildGoModule }:
```

Comment:
You could replace ``stdenv`` with ``lib``.

Reply:
Applied, thanks